BUILD_macos.md

```diff
@@ -70,6 +70,12 @@ PRs with corrections and updates are welcome!
   brew install libfido2
   ```
 
+* To install `yarn` for building the UI
+  * Install Node.js from the [Node.js Releases/Download Page](https://nodejs.org/en/download/)
```

Comment:
I've never installed yarn on my mac with any method other than homebrew and never had a problem. (I don't often build Teleport Connect though, which is where the v16 requirement comes from).

Anyway, what do you think about mentioning homebrew first? IME that's what people expect as a first option on mac.

Reply:
This should be ok but `nodejs` currently installs v19, so we'll want to specify `node@16`. In the near future we can update it to be `node@18`.

Reply:
+1 for Homebrew as a suggested path.